Ladybird/WebContent/CMakeLists.txt

```diff
@@ -70,6 +72,10 @@ else()
     add_executable(WebContent main.cpp)
 endif()
 
+if (HAS_FONTCONFIG)
+    target_link_libraries(webcontent PUBLIC fontconfig)
```

Comment:
This should be linked privately to LibGfx, not publicly to the executables.

Reply:
Also we want to link against `Fontconfig::Fontconfig`, the imported target, that includes all compile and linker flags to use fontconfig. 

https://cmake.org/cmake/help/latest/module/FindFontconfig.html

Reply:
Also probably only want to add `USE_FONTCONFIG` as a private `target_compile_definitions` on LibGfx, rather than defining it for everyone.

Reply:
I expect fontconfig to be used in a few more places actually, so we'll see..